config-templates/insolar_1.yaml

```diff
@@ -44,7 +44,8 @@ ledger:
     postprocessbackupcmd:
     - bash
     - -c
-    - ./bin/backupmanager daemon-merge -a http://localhost:8099 -g -n $INSOLAR_CURRENT_BACKUP_DIR/incr.bkp >> ./backupmanager.log 2>&1 && touch $INSOLAR_CURRENT_BACKUP_DIR/BACKUPED
+    - echo $$ > /tmp/heavy/backup.pid ; sleep 1; CMD="rsync -avz --delete ./data/* ./heavy_backup" ; while $CMD | grep -q "(MANIFEST\|\.sst)$"; do :; done ; touch $INSOLAR_CURRENT_BACKUP_DIR/BACKUPED
+
```

Comment:
why `sleep 1`?

Reply:
yes it may work very quick and will never be caught to kill

Reply:
did you consider to kill `rsync` by the process name as it was implemented before?

Reply:
the same story, it may work very quick. Before, we had daemon which run forever, and that is why there were no problems

Reply:
IIRC we killed `backupmanager`, not a `backupdaemon`, and it used to run exactly the same way as `rsync` does.

Reply:
Ok